tide/core/Configuration.h

```diff
@@ -84,6 +84,12 @@ class Configuration
     /** Get the filename passed to the constructor. */
     const QString& getFilename() const;
 
+    /** Get the number of vertical bezels. */
+    int getBezelsVertical() const;
```

Comment:
The same remark applies here and everywhere in fact, I think it should be getBezelsPerScreenX as is it not a global count of bezels